# numba <0.53.0 restriction were made due to the bug in umap-learn, see:
# https://github.com/theislab/cellrank/commit/e5c6191ac50401ecda5c46ff5e9e92a86d6f3769#diff-2c9c11d26b09b8afde329980309d967121543a456e4592c76886a20b5cf56c90L8-L9
# Since this fixed in scVelo-0.2.3-foss-2021a.eb, see: 
# https://github.com/easybuilders/easybuild-easyconfigs/pull/13398/files#diff-db270fae41d98bc2f577480759252029490fb0d9518d59b8fd3b0f13b0e2b820R1-R17
# This limit can be safely fifted
# July 7th 2021 by B. Hajgato (UGent)
diff -ru cellrank-1.4.0.orig/cellrank.egg-info/requires.txt cellrank-1.4.0/cellrank.egg-info/requires.txt
--- cellrank-1.4.0.orig/cellrank.egg-info/requires.txt	2021-06-30 17:48:49.000000000 +0200
+++ cellrank-1.4.0/cellrank.egg-info/requires.txt	2021-07-09 20:15:21.756925337 +0200
@@ -2,7 +2,7 @@
 joblib>=0.13.1
 matplotlib>=3.3.0
 networkx>=2.2
-numba<0.53.0,>=0.51.0
+numba>=0.51.0
 numpy>=1.2.0
 pandas>=1.2.0
 pygam>=0.8.0
diff -ru cellrank-1.4.0.orig/requirements.txt cellrank-1.4.0/requirements.txt
--- cellrank-1.4.0.orig/requirements.txt	2021-06-30 17:48:33.000000000 +0200
+++ cellrank-1.4.0/requirements.txt	2021-07-09 20:15:32.333795389 +0200
@@ -2,7 +2,7 @@
 joblib>=0.13.1
 matplotlib>=3.3.0
 networkx>=2.2
-numba>=0.51.0,<0.53.0
+numba>=0.51.0
 numpy>=1.2.0
 pandas>=1.2.0
 pygam>=0.8.0
